terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 1: 26306 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 2: 26308 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 3: 26310 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 4: 26312 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 5: 26314 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-5-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 6: 26316 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 7: 26318 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 8: 26320 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 9: 26322 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 10: 26324 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-6-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 11: 26326 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 12: 26328 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 13: 26330 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 14: 26332 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 15: 26334 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-7-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 16: 26336 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 17: 26338 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 18: 26340 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 19: 26342 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 20: 26344 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-8-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 21: 26346 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 22: 26348 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 23: 26350 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 24: 26352 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 25: 26354 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-9-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 26: 26356 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 27: 26358 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 28: 26360 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 29: 26362 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 30: 26364 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-10-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 31: 26366 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 32: 26368 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 33: 26370 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 34: 26372 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 35: 26374 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-11-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 36: 26376 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 37: 26378 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 38: 26380 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 39: 26382 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 40: 26384 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-12-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 41: 26386 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 42: 26388 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 43: 26390 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 44: 26392 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 45: 26394 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-13-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 46: 26396 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 47: 26398 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 48: 26400 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 49: 26402 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 50: 26404 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-14-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 51: 26406 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 52: 26408 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 53: 26410 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 54: 26412 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 55: 26414 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-15-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 56: 26416 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 57: 26418 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 58: 26420 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 59: 26422 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 60: 26424 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-16-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 61: 26426 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 62: 26428 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 63: 26430 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 64: 26432 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 65: 26434 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-17-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 66: 26436 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 67: 26438 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 68: 26440 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 69: 26442 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 70: 26444 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-18-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 71: 26446 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 72: 26448 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 73: 26450 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 74: 26452 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 75: 26454 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-19-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 76: 26456 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 77: 26458 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 78: 26460 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 79: 26462 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 80: 26464 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-20-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 81: 26466 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 82: 26468 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 83: 26470 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 84: 26472 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 85: 26474 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-21-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 86: 26476 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-4-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'domains/neg/gripper/gripper-1-pos-a.pddl'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 87: 26478 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-1-pos-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'domains/neg/gripper/gripper-1-neg-d.pddl'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 88: 26480 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-1-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'domains/neg/gripper/gripper-1-neg-b.pddl'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 89: 26482 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-1-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 90: 26484 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-2-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 91: 26486 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-3-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 92: 26488 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-2-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 93: 26490 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-4-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 94: 26492 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-3-neg-b.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 95: 26494 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-2-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'domains/neg/gripper/gripper-1-neg-c.pddl'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 96: 26496 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-1-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 97: 26498 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-2-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 98: 26500 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-4-neg-a.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 99: 26502 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-4-neg-d.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 100: 26504 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-3-neg-c.pddl $1
terminate called after throwing an instance of 'std::runtime_error'
  what():  Failed to open file 'experiments/neg/ijcai19/gripper/29.neg.program'
./scripts/neg/ijcai19/gripper-val/gripper-NEG-2-2-a-val.sh: line 101: 26506 Aborted                 (core dumped) ./universal-planning-validator/validator/validate.bin -p domains/neg/gripper/domain.pddl domains/neg/gripper/gripper-3-neg-a.pddl $1
